pkg/apis/networking/v1/types.go

```diff
@@ -210,6 +211,10 @@ type APIServerConnConfig struct {
 type RemoteClusterStatus struct {
 	// Conditions is an array of current cluster conditions.
 	Conditions []ClusterCondition `json:"conditions"`
+	// A globally unique identifier. Use for validating.
+	// Generated by directly using the corresponding remote cluster's namespace "kube-system" uid
+	// types.UID is also universally unique identifiers (also known as UUIDs).
+	UUID types.UID `json:"uuid"`
```

Comment:
I think `string` is more general here?

Reply:
types.UID is actually a type definition of string, which means it is convenient to convert from string or convert to string, what's more, it is used in kubernetes, i think directly use it is more understandable.